docs/user-guide/review.md

```diff
@@ -0,0 +1,48 @@
+# Crawl Review
+
+The Crawl Review page provides a streamlined interface for assessing the capture quality of pages within an archived item using the heuristics collected during crawl analysis.
+
+Crawls can only be reviewed once [crawl analysis](archived-items.md#crawl-analysis) has been run. If multiple analysis runs have been completed, the page analysis heuristics will be used from the selected analysis run, available beside the archived item name.
+
+## Heuristics
+
+Crawl analysis generates comparisons across three heuristics that can indicate which pages may be the most problematic.
+
+### Screenshot Comparison
+
+Screenshots taken during crawl analysis are compared with the page thumbnail created during crawling. Screenshots are compared using [Pixelmatch](https://github.com/mapbox/pixelmatch).
+
+Discrepancies between crawl and replay screenshots may occur because resources aren't loaded or rendered properly (usually indicating a replay issue).
+
+!!! Tip "Caveats"
+    If many similar pages exhibit similarly poor screenshot comparison scores but look fine in the replay tab, it may be because of page loading time not being long enough during analysis.
+
+    Some websites may take more time to load than others, including on replay! If the page wasn't given enough time to load during crawl analysis — because crawl analysis uses the same workflow limit settings as crawling — increasing the [_Delay After Page Load_ workflow setting](workflow-setup.md#delay-after-page-load) may yield better screenshot analysis scores, at the cost of extra execution time.
+
+### Extracted Text Comparison
+
+Text extracted during crawl analysis is compared to the extracted text created during crawling. Extracted text is compared on the basis of [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance).
```

Comment:
```suggestion
Text extracted during crawl analysis is compared to the text extracted during crawling. Text is compared on the basis of [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance).
```

Reply:
Hmm, not sure I like this change, I think we need to fit "extracted" in at _least_ 3 more times before I'm truly happy with it.  ;)